doc/rtd/reference/datasources/nocloud.rst

```diff
@@ -39,7 +39,10 @@ In a similar fashion, configuration files can be provided to cloud-init using a
 custom webserver at a URL dictated by kernel commandline arguments or SMBIOS
 serial number. This argument might look like: ::
 
-  ds=nocloud s=http://10.42.42.42/cloud-init/configs/
+  ds=nocloud-net;s=http://10.42.42.42/cloud-init/configs/
+
+.. note::
+   When supplementing kernel parameters in Grub's boot menu take care to single-quote this full value to avoid Grub interpreting the semi-colon as a reserved word. See: `Grub Quoting`_
```

Comment:
```suggestion
   When supplementing kernel parameters in GRUB's boot menu take care to single-quote this full value to avoid GRUB interpreting the semi-colon as a reserved word. See: `GRUB quoting`_
```